docs/source/development/index.rst

```diff
@@ -41,4 +39,5 @@ A typical development workflow might look like the following:
     release-workflow
     style-guide/style-guide
     tools/index
+    CDFs/index
     poetry
```

Comment:
Since we now have quite a growing list here, maybe we should just put these in alphabetical order?